packages/node_modules/@ciscospark/plugin-memberships/src/memberships.js

```diff
@@ -24,6 +38,148 @@ import {SparkPlugin, Page} from '@ciscospark/spark-core';
  * @name Memberships
  */
 const Memberships = SparkPlugin.extend({
+  /**
+   * Connect to the web socket to listen to incoming messages.
+   * @returns {Promise}
+   */
+  listen() {
+    // Create a common envelope that we will wrap all events in
+    this.eventEnvelope = {
+      resource: 'memberships',
+      // orgId -- TODO get from this.spark.internal.user
+      // createdBy -- TODO get from this.spark.internal.user
+      ownedBy: 'creator',
+      status: 'active',
+      created: new Date().toISOString(),
+      data: {}
+    };
+    // Register to listen to events
+    return this.spark.internal.mercury.connect()
+      .then(() => this.listenTo(this.spark.internal.mercury, CONVERSATION_ACTIVITY,
+        (event) => this.onWebexApiEvent(event)));
+  },
+
+  /**
+   * Trigger a membership related events.
+   * @param {Object} event
+   * @returns {undefined}
+   */
+  onWebexApiEvent(event) {
+    const {activity} = event.data;
+    console.log(activity);
```

Comment:
This should be removed. You can use `debug(activity)` if it's valuable for debugging.

Reply:
Agree.

Reply:
Removed